src/de.pengutronix.rauc.Installer.xml

```diff
@@ -36,6 +36,19 @@
       <arg name="compatible" type="s" direction="out" />
       <arg name="version" type="s" direction="out" />
     </method>
+ 
+    <!--
+         InspectBundle: D-Bus variant of rauc info <bundle>
+         @source: Path to the queried bundle.
+         @args: Additional arguments to pass
+         @bundle_info: string variant dict of bundle information
+    -->
+    <method name="InspectBundle">
+      <arg name="source" type="s" direction="in" />
+      <arg name="args" type="a{sv}" direction="in"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In1" value="QVariantMap"/>
+      <arg name="bundle_info" type="a{sv}" direction="out"/>
```

Comment:
Just use `info`, as we may also want to return HTTP headers from the server in the future.